aiogram/types/base.py

```diff
@@ -117,7 +118,7 @@ def props_aliases(self) -> typing.Dict[str, str]:
         return getattr(self, ALIASES_ATTR_NAME, {})
 
     @property
-    def values(self):
+    def values(self) -> T:
```

Comment:
values is not accessible via this property (deprecated)
That's mean need to mark this method as deprecated.

About typing:
```suggestion
    def values(self) -> typing.Tuple[str]:
```